setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup(
     name='smbprotocol',
-    version='1.0.2.dev0',
+    version='1.0.3',
```

Comment:
Best to leave this as is, technically the next release will now be `1.1.0` as this is a new feature but I deal with that when packaging a new version.